### PR TITLE
✨ Add Garden page

### LINKED ISSUE #5: "Garden" page
#### Description

On this page the user  finds a garden full of Options. 

- The `<Link>NewPlant</Link>`brings the user to the "NewPlant" page to create a new plant for his pocket-garden.

image

- The `<Link>PlantList</Link>`brings the user to the "PlantList" page to see all his created plans for his pocket-garden.

image

- The `<Link>NewTask</Link>`brings the user to the "NewTask" page to create a new task for his pocket-garden.

image

- The `<Link>Gardening</Link>`brings the user to the "Gardening" page to see all present tasks for his pocket-garden.

image

#### Screenshots

image

#### Tasks

`Components`

- [x] `<Link>NewPlant</Link>`
- [x] `<Link>PlantList</Link>`
- [x] `<Link>NewTask</Link>`
- [x] `<Link>Gardening</Link>`
- [x] Implement Subtitle `component`

### PR DESCRIPTION
**Pull request by MaxMuecket**
fixes #5

- Add leaf.png asset
- Add PageLink component
- Add Garden page
- Add path for Garden page

### CODE REVIEW

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀 It would be nice to see that result.

**Review by lajanzen — APPROVED** [review]
🚀

File: src/app/globals.css [hunk 1]
```diff
@@ -28,5 +28,5 @@ body {
   -moz-osx-font-smoothing: grayscale;
   font-family: var(--font-family);
   color: var(--primary-text-color);
-  background-color: var(--background-color);
+  background-color: #fff;
```
Comment (phlgr) [inline src/app/globals.css:31]: Why did you remove the variable

File: src/app/pages/GardenPage/GardenPage.module.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  font-size: 3.438rem;
+  font-weight: 300;
+  margin: 0;
+  padding: 1rem;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: 1rem 11.75rem 0.8rem 12.5rem;
+  grid-template-rows: 1.5rem 11.65rem 7.65rem 13.25rem;
```
Comment (phlgr) [inline src/app/pages/GardenPage/GardenPage.module.css:17]: Quite complicated could this also be solved with percentages?

File: src/app/pages/LandingPage/LandingPage.tsx [hunk 3]
```diff
@@ -5,8 +5,8 @@ import styles from './LandingPage.module.css';
 function LandingPage(): JSX.Element {
   return (
     <div className={styles.container}>
-      <a className={styles.title} href="#">
-        Pocket-
+      <a className={styles.title} href="garden">
+        Pocket
         <br />
```
Comment (phlgr) [inline src/app/pages/LandingPage/LandingPage.tsx:10]: Please don't use a br as this is purely visual this should be solved with css
